api/UIEvent.json

```
@@ -385,10 +385,10 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "≤4"
+              "version_added": "3"
```

```suggestion
              "version_added": "1"
```

Source: https://github.com/WebKit/WebKit/blob/7659cc58c0a8c7c04bc854486cb8355a13c008dd/WebCore/khtml/ecma/kjs_events.cpp#L250